### PR TITLE
Add 'Reported Skills' tab in Admin section #2187

### LINKED ISSUE #2187: Add 'Reported Skills' tab on Admin section
**Actual Behaviour**
Not present.

**Expected Behaviour**
* Add 'Reported Skills' tab on Admin section

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #2187

Changes: Added 'Reported Skills' tab in the Admin section

Demo Link: https://pr-2374-fossasia-susi-web-chat.surge.sh

Screenshots of the change: 

Screenshot 2019-06-26 at 8 52 13 PM

Screenshot 2019-06-26 at 8 52 24 PM

Screenshot 2019-06-26 at 8 52 31 PM

### CODE REVIEW

**Comment by fragm3** [comment]
Updated:

Screenshot 2019-06-25 at 6 01 38 PM

**Comment by fragm3** [comment]
@akshatnitd made the changes:

Screenshot 2019-06-26 at 2 25 22 AM

**Comment by fragm3** [comment]
@akshatnitd @harsh-jindal The delete API is not yet implemented on the backend, will implement it soon and make the changes on the backend as well. Same for segregating the skills feedback

**Comment by akshatnitd** [comment]
@fragm3 The action is to delete the skill. We don't need to delete the feedback.

**Comment by fragm3** [comment]
@akshatnitd @harsh-jindal made the changes, screenshot in PR description updated

**Comment by harsh-jindal** [comment]
@fragm3 Unable to see reported skills in surge link.
Screenshot 2019-06-26 at 11 21 24 PM

**Comment by fragm3** [comment]
@harsh-jindal It will work once PR on the server side is merged.

File: src/apis/index.js [hunk 1]
```diff
@@ -674,3 +674,9 @@ export function fetchRevertingCommitInformation(payload) {
   const { url } = payload;
   return ajax.get(url, {});
 }
+
+// ReportedSkills
+export function fetchReportedSkills() {
+  const url = `http://127.0.0.1:4000/${CMS_API_PREFIX}/getReportSkill.json`;
```
Comment (akshatnitd) [inline src/apis/index.js]: Correct the url

**Comment by fragm3** [comment]
@akshatnitd made the change

File: src/components/Admin/ListSkills/ListSkills.js [hunk 2]
```diff
@@ -492,12 +509,84 @@ class ListSkills extends React.Component {
       },
     ];
 
+    let reportedColumns = [
```
Comment (akshatnitd) [inline src/components/Admin/ListSkills/ListSkills.js]: Declare it outside the class definition.

Comment (akshatnitd) [reply src/components/Admin/ListSkills/ListSkills.js]: Probably in a constants file. Do it for similar tables as well.

**Comment by fragm3** [comment]
@akshatnitd made the changes.

**Pull request merged**